public/scripts/timeline.js

```diff
@@ -968,6 +992,133 @@ function getStats(){
                     $(".statsWrapper").show();
                     $(".getSummary").show();
 
+                    //$(".approxTimeShowingPTag").show(800).delay(5000).fadeOut();
+                    $(".modal-backdrop").remove();
+                    $('#serverStreamingModal').modal('hide');
+
+                }catch(err){
+                    alert($.trim(data));
+                    $('#serverStreamingModal .logsContent').empty();
+                      $('#serverStreamingModal').modal('hide');
+                    $(".statsWrapper").hide();
+                    $(".tabContentWrapper").hide();
+                }
+            },
+            error: function( data, textStatus, jqXHR,err) {
+              // $("#busy-loader").hide();
+              // $('#serverStreamingModal .logsContent').empty();
+              //   $('#serverStreamingModal').mo dal('hide');
+               var errMsg = "Some problem fetching the response, Please refresh and try again.";   
+              console.log("readyState: " + jqXHR.readyState);
+              console.log("responseText: "+ jqXHR.responseText);
+              console.log("status: " + jqXHR.status);
+              console.log("text status: " + textStatus);
+              console.log("error: " + err);

```

Comment:
If these logs are useful in production then keep them here otherwise if these are for debugging then decide whether or not you should remove it.